solana-library/system_instruction.sol

```diff
@@ -0,0 +1,297 @@
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
```suggestion
// This code is provided as-is. It is not audited and any use is at your own risk.

```

Reply:
I wrote my own version before seeing your suggestion. Please, check if you agree with my phrasing.